rflx/pyrflx/message.py

```diff
@@ -352,3 +352,29 @@ def __simplified(self, expr: Expr) -> Expr:
         }
 
         return expr.simplified(field_values).simplified(self.__type_literals)
+
+    def parse_from_bitstring(self, msg: str) -> None:
```

Comment:
Why not directly `parse_from_bytes`? The user should not have to convert this into a bitstring manually.